src/PhpSpreadsheet/Shared/StringHelper.php

```diff
@@ -478,7 +478,7 @@ public static function countCharacters($textValue, $encoding = 'UTF-8')
      */
     public static function substring($textValue, $offset, $length = 0)
```

Comment:
Would it be easier to enforce this with scalar type-hinting for the method? We're not using `strict_types`, so this would be coercive
```
public static function substring(string $textValue, int $offset, int $length = 0)
```